src/sass/_mixins.scss

```diff
@@ -62,6 +62,15 @@
   // scss-lint:enable NameFormat
 }
 
+@mixin uppercase($letterSpacing: null){
```

Comment:
I like the concept I don't like the name but I can't think about anything better than that :(